.github/workflows/ci.yml

```diff
@@ -378,6 +392,10 @@ jobs:
           sha256sum jq-* > sha256sum.txt
           gh release create "$TAG_NAME" --draft --title "jq ${TAG_NAME#jq-}" --generate-notes
           gh release upload "$TAG_NAME" --clobber jq-* sha256sum.txt
+      - name: attest-build-provenance
```

Comment:
Could you clalify the step name like others? You can commit squash.

Reply:
I'm not 100% sure what you mean but does this work?

`- name: Release attest-build-provenance`

https://github.com/jqlang/jq/pull/3225/files#diff-b803fcb7f17ed9235f1e5cb1fcd2f5d3b2838429d4368ae4c57ce4436577f03fR395

Reply:
How about `Generate signed attestations`

Reply:
done